3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -1,40 +1,39 @@
 'use strict';
-/*------------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/tree/main/3-UsingAPIs/Week1#exercise-2-is-it-a-double-digit-number
 
-Complete the function called `checkDoubleDigits` such that:
-
-- It takes one argument: a number
-- It returns a `new Promise`.
-- If the number between 10 and 99 it should resolve to the string
-  "This is a double digit number!".
-- For any other number it should reject with an an Error object containing: 
-  "Expected a double digit number but got `number`", where `number` is the 
-  number that was passed as an argument.
-------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      if (number >= 10 && number < 100) {
+        resolve('This is a double digit number!');
+      } else {
+        reject(
+          new Error(
+            `Expected a double digit number but got ${number}, where ${number} is the number that was passed as an argument`
+          )
```

Comment:
The part after the comma was intended as an explanation for the developer, not as something to be included literally.

```suggestion
          new Error(
            `Expected a double digit number but got ${number}`
          )
```